### PR TITLE
LPS-74902 Deprecate util java usages

### PR DESCRIPTION
**Pull request by tomwang2011**

### CODE REVIEW

**Comment by tomwang2011** [comment]
ci:reopen poshi runner unit test shouldn't be affected by this pull

**Comment by tomwang2011** [comment]
The SpringExtenderDependencyManagerTest failures should not be related to this pull, thanks!

**Comment by shuyangzhou** [comment]
ci:retest rerun anyway to make sure the springextender failure did not hide others.

**Comment by tomwang2011** [comment]
Looks like CI is having memory size issues, the failures doesn't seem to be related to this, thanks!

File: portal-impl/bnd.bnd [hunk 1]
```diff
@@ -78,8 +78,13 @@ Export-Package:\
 Import-Package: *
 Include-Resource:\
 	classes,\
+	@../tmp/lib-pre/com.liferay.petra.content.jar,\
```
Comment (shuyangzhou) [inline portal-impl/bnd.bnd]: @tomwang2011 Why are we merging them back into portal-impl?

File: modules/core/portal-bootstrap/build.gradle [hunk 2]
```diff
@@ -85,6 +85,10 @@ dependencies {
 	provided group: "org.springframework", name: "spring-beans", version: "4.1.9.RELEASE"
 	provided group: "org.springframework", name: "spring-context", version: "4.1.9.RELEASE"
 	provided group: "org.springframework", name: "spring-core", version: "4.1.9.RELEASE"
+	provided project(":apps:foundation:petra:petra-content")
```
Comment (shuyangzhou) [inline modules/core/portal-bootstrap/build.gradle]: @tomwang2011 they are all released, right? Why we do project dependencies here?

**Pull request closed without merging**